packages/schema/src/config/app.ts

```diff
@@ -274,7 +274,9 @@ export default defineUntypedSchema({
      * Customize Nuxt Nuxt SpaLoader element attributes.
      * @type {typeof import('@unhead/schema').HtmlAttributes}
      */
-    spaLoaderAttrs: {},
+    spaLoaderAttrs: {
```

Comment:
maybe, better?
```
    spaLoaderAttrs: {
<<<<<<< HEAD
      $resolve: (val: undefined | null | Record<string, unknown>) => {
        return defu(val, {
          id: val?.id ?? '__nuxt-spa-loader',
        })
      },
=======
      id: '__nuxt-loader',
>>>>>>> f4106a79061f41753ee9caff028d8755272634af
    },
  },
```

Reply:
not sure I understand

Reply:
So, if I pass

spaLoaderAttrs: { class: "some-class" }
it will ultimately result in

{ class: "some-class", id: "__nuxt-loader" }
right?